xbmc/addons/gui/skin/SkinTimerManager.cpp

```diff
@@ -58,8 +58,8 @@ void CSkinTimerManager::LoadTimerInternal(const TiXmlElement* node)
   INFO::InfoPtr resetInfo{nullptr};
   INFO::InfoPtr stopInfo{nullptr};
   std::string timerName = node->FirstChild("name")->FirstChild()->Value();
-  std::string startAction;
-  std::string stopAction;
+  CGUIAction startActions;
+  CGUIAction stopActions;
```

Comment:
Any reason for not declaring the vars next to where they are needed? Declaring all vars at the begin of the function is so ... C-like. ;-)

Reply:
personal preference, I prefer to have all the parameters required for the timer construction grouped together instead of constructing each of them as we go

Reply:
But if you don't need them, this is waste of resources.

Reply:
You've a point :)
Done